db/schema.rb

```diff
@@ -0,0 +1,31 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 0) do
+
+  create_table "ht_users", primary_key: "userid", id: :string, limit: 256, default: "", options: "ENGINE=MyISAM DEFAULT CHARSET=latin1", force: :cascade do |t|
+    t.string "displayname", limit: 128
+    t.string "email", limit: 128
+    t.string "activitycontact", limit: 128
+    t.string "approver", limit: 128
```

Comment:
If the userid can be 256, don't these need to be as well?

Reply:
userid could be a shib persistent ID which is probably why it's longer. This schema is just copied from production, and changes to it are out of scope for this project.